.travis.yml

```diff
@@ -9,9 +9,11 @@ branches:
 language: go
 go:
   - "tip"
-  - "1.8"
-  - "1.9"
-  - "1.10"
+  - "1.x"
```

Comment:
This will use only latest golang version, would be nice to keep latest versions supporting go mod (1.10, 1.11, 1.12) to check if it compiles and works correctly on these versions

Reply:
I have updated the list to keep the most recent 2 versions (which also support Go modules). Does that sound ok?

Reply:
Ok, thanks

Reply:
xref: https://github.com/golang/go/issues/30446#issuecomment-468028807

So, every dependency also needs to support Go mod 1.11

Reply:
Filed a pr with the mssql server driver https://github.com/denisenkom/go-mssqldb/pull/503